### PR TITLE
Correct spelling mistakes

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Perfect documentation.
**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/api/testing/unstructured_test.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func doRoundTrip(t *testing.T, internalVersion schema.GroupVersion, externalVers
 	fuzzer.FuzzerFor(FuzzerFuncs, rand.NewSource(seed), legacyscheme.Codecs).
 		// We are explicitly overwriting custom fuzzing functions, to ensure
 		// that InitContainers and their statuses are not generated. This is
-		// because in thise test we are simply doing json operations, in which
+		// because in these test we are simply doing json operations, in which
```
Comment (liggitt) [inline pkg/api/testing/unstructured_test.go]: this

File: pkg/controller/nodeipam/ipam/range_allocator.go [hunk 2]
```diff
@@ -61,7 +61,7 @@ type rangeAllocator struct {
 	// This increases a throughput of CIDR assignment by not blocking on long operations.
 	nodeCIDRUpdateChannel chan nodeReservedCIDRs
 	recorder              record.EventRecorder
-	// Keep a set of nodes that are currectly being processed to avoid races in CIDR allocation
+	// Keep a set of nodes that are correctly being processed to avoid races in CIDR allocation
```
Comment (liggitt) [inline pkg/controller/nodeipam/ipam/range_allocator.go]: currently

File: pkg/kubelet/eviction/eviction_manager_test.go [hunk 3]
```diff
@@ -927,7 +927,7 @@ func TestNodeReclaimFuncs(t *testing.T) {
 
 	// no image gc should have occurred
 	if diskGC.imageGCInvoked || diskGC.containerGCInvoked {
-		t.Errorf("Manager chose to perform image gc when it was not neeed")
+		t.Errorf("Manager chose to perform image gc when it was not need")
```
Comment (liggitt) [inline pkg/kubelet/eviction/eviction_manager_test.go]: needed

File: pkg/kubelet/eviction/eviction_manager_test.go [hunk 4]
```diff
@@ -950,7 +950,7 @@ func TestNodeReclaimFuncs(t *testing.T) {
...
```
Comment (liggitt) [inline pkg/kubelet/eviction/eviction_manager_test.go]: needed

Comment (yuxiaobo96) [reply pkg/kubelet/eviction/eviction_manager_test.go]: OK.

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

**Pull request merged**